_features/fresh.html

```diff
@@ -2,6 +2,14 @@
 title: Fresh
 slug: fresh
 description: Always up-to-date.
+usecases:
+- As a web application developer, I want to deliver updates seamlessly, without requiring
+  user intervention and approval.
+- As a user, I don't want to wait for the new version to be installed; I always
+  want the latest version available.
 technologies:
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>When users visit your web app, they are not prompted with a request to update,
+they simply see yout brand new UI with all your awesome functionality. As web apps, progressive web apps
+behave the same way out of the box but provide you with the means to fully customize the
+life cycle of your app.</p>
```

Comment:
> I'm trying to explain what is fresh in the context of a progressive web application. In the original blog post, fresh refer to the fact that a web site is always up to date. The user is never prompted about installing or updating because that is not the model of the Web.

Hmm, but that blog post defines *fresh* as "transparently always up-to-date thanks to the Service Worker update process." It doesn't reference a model of the web.

> Progressive web applications, as web applications behave this way out of the box: you update the server, the user gets the new version.

Ok, that makes sense, and it's reasonable to explain how silent updates using Service Workers are consistent with updates on the web that don't use Service Workers (even if they original blog post doesn't make that point). Although that isn't actually a *benefit* of progressive web apps, since it doesn't make them any better than regular web apps.

> I think trying to communicate this is enough.

Erm, does this mean that you no longer want to reference the ability to customize the application lifecycle? That ability actually seems useful to me, especially since it's the way in which progressive web apps are different from regular web apps. Even though, ironically, customization in this case means the ability to preserve *stale* installations or create inconsistent ones.